yuca/ca/common.py

```diff
@@ -122,8 +120,12 @@ def initialize_id_layer(self):
 
         padding = (self.id_dim - 1) // 2 
 
+        groups = 1 if self.internal_channels % self.external_channels \
```

Comment:
It's necessary to match groups and channels in the `id_layer` to yield identity convolutions. We should probably print a warning when a system is initialized with a mis-match.

Reply:
Warning adding when groups != channels